parts/linux/cloud-init/artifacts/cis.sh

```diff
@@ -2,10 +2,15 @@
 
 assignRootPW() {
   if grep '^root:[!*]:' /etc/shadow; then
+    VERSION=$(grep DISTRIB_RELEASE /etc/*-release| cut -f 2 -d "=")
     SALT=$(openssl rand -base64 5)
     SECRET=$(openssl rand -base64 37)
-    CMD="import crypt, getpass, pwd; print crypt.crypt('$SECRET', '\$6\$$SALT\$')"
-    HASH=$(python -c "$CMD")
+    CMD="import crypt, getpass, pwd; print(crypt.crypt('$SECRET', '\$6\$$SALT\$'))"
+    if [[ "${VERSION}" == "22.04" ]]; then
+      HASH=$(python3 -c "$CMD")
```

Comment:
suggest to always use python3 and ensure it's installed in vhd, I thought it was already just not symlinked to `python`